tests/unit/test_transformer.py

```diff
@@ -59,6 +59,13 @@ def transformer(sagemaker_session):
                        volume_kms_key=KMS_KEY_ID)
 
 
+def test_delete_model(sagemaker_session):
+    transformer = Transformer(MODEL_NAME, INSTANCE_COUNT, INSTANCE_TYPE, output_path=OUTPUT_PATH,
```

Comment:
Feel free to remove the optional parameters.

https://github.com/aws/sagemaker-python-sdk/blob/757c8ab8a48000344e37bc0683348e9060ffde17/src/sagemaker/transformer.py#L24

Reply:
Do you need the output_path or volume_kms_key?

Reply:
This was updated in previous commit.